srcpkgs/Lidarr/template

```diff
@@ -0,0 +1,41 @@
+# Template file for 'Lidarr'
+pkgname=Lidarr
+version=0.5.0.583
+revision=1
+make_dirs="
```

Comment:
make_dirs= go below the variables block

Reply:
I wasn't sure exactly what you meant. Is this it?

Reply:
yes.